doc/ChangeLog

```diff
@@ -1,5 +1,29 @@
 ===============================================================
 
+Tag name:
+Originator(s): manderberg
+Date: Dec 1, 2021
```

Comment:
Should update the date ?

Reply:
Updated ChangeLog to reflect new date (Jan 19, 2022) and pushed it up to my carma_kappa_test branch.